frontend/src/routes/home/Home.tsx

```diff
@@ -108,10 +110,13 @@ function Home(props) {
           <div className="program-section">
             <div className="next">urmează:</div>
             {getNext2Days().map((dayN) => (
-              <OneDayProgram dayNumber={dayN} programs={program} key={dayN} />
+              <OneDayProgram dayNumber={dayN} programs={program} key={uuid()} />
```

Comment:
Can a increment be used here and avoid the includsion of uuid library?

Reply:
sure